src/Steps/Release/RewriteReleaseBranches.php

```diff
@@ -91,13 +91,13 @@ protected function branchLibrary(OutputInterface $output, LibraryRelease $librar
         if (empty($target) || $target === $currentBranch) {
             $this->log(
                 $output,
-                "Releasing library <info>{$libraryName}</info> from branch <info>{$currentBranch}</info>"
+                "Branching module <info>{$libraryName}</info> from branch <info>{$currentBranch}</info>"
```

Comment:
I think I understand why they used "Releasing" here now - it seems to be an informational thing ("this module is already pointing at the branch we'll be releasing it from".) Based on this, I think the language should still vary from the other case, but be a bit more explicit.

```suggestion
                "Module <info>{$libraryName}</info> already on correct branch (<info>{$currentBranch}</info>)"
```